example-ideal.txt

```diff
@@ -0,0 +1,20 @@
+// Implementation example for empty picobot maze
+
+state leftTraverse {
+	move left;
```

Comment:
Any particular reason why the `move` statement requires a semicolon, but the `else` statement does not?